ld/armv7a7-imx6ull.ldt

```diff
@@ -49,6 +57,26 @@ REGION_ALIAS("HEAP", m_ocram_low);
 REGION_ALIAS("STACK", m_ocram_low);
 REGION_ALIAS("PLO_IMAGE", m_ocram_high);
 REGION_ALIAS("BSS", m_ocram_high);
+REGION_ALIAS("NAND", m_ddr);
+
+SECTIONS
+{
+	.nand ADDR_NAND (NOLOAD) :
+	{
+		. = ALIGN(4);
```

Comment:
Aliment to 4 is enough for this DMA?

Reply:
I think so, not 100% sure though  - I'll try to test it.